lib/ipaddr_helpers.rb

```diff
@@ -244,4 +245,9 @@ def supernets
         end
       end
     end
+
+    def ping?
```

Comment:
I'm against this.. we should not add new methods to monkey patched IPAddr unless really necessary.